app/lib/webrtc.js

```
@@ -0,0 +1,233 @@
+}
+
+/**
+ * Adds media stream to the peer connection
+ * @param {RTCPeerConnection} peerConnection - The RTCPeerConnection object
+ * @param {MediaStream} stream - The media stream to add
+ */
+export function addMediaStream(peerConnection, stream) {
+  stream.getTracks().forEach(track => {
+    peerConnection.addTrack(track, stream);
+    console.log('Track added to peer connection:', track.kind);
+  });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)